<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SmartRadio - Secure Access</string>
  </property>
  <widget class="QWidget" name="mainContainer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QLabel" name="bgLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="welcomeTitle">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>500</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string>welcom to smart radio</string>
    </property>
   </widget>
   <widget class="QLabel" name="welcomeSubtitle">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>500</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>THE NEW RADIO GENERATION</string>
    </property>
   </widget>
   <widget class="QFrame" name="loginCard">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>340</width>
      <height>500</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="logoLabel">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>120</width>
       <height>120</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="usernameEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>260</width>
       <height>40</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passwordEdit">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>260</width>
       <height>40</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="loginBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>310</y>
       <width>260</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>log-in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signupBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>365</y>
       <width>260</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>sign-up</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancelBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>420</y>
       <width>260</width>
       <height>45</height>
      </rect>
     </property>
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
    <widget class="QLabel" name="forgotLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>450</y>
       <width>260</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>forgot password ?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="arduinoLogo">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>470</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cppLogo">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>470</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
